### ISSUE TITLE
VueJS

### ISSUE THREAD

**Issue by ronaldohoch** [dimsemenov/PhotoSwipe#1654]
First I want to thank you for this plugin, it's wonderful <3
Secondly, how can I import it into Vue2

I can't make it work with my Vue website.

Thank you.

**Comment by Radla** [comment]
maybe this helps? https://github.com/dimsemenov/PhotoSwipe/issues/1649#issuecomment-536259111

**Comment by mcrider** [comment]
I have it working with https://www.npmjs.com/package/vue-simple-photoswipe.

**Comment by ronaldohoch** [comment]
@mcrider Have you made a production build? Two galeries in same page?

**Comment by mcrider** [comment]
Yep! E.g. https://www.cressyhouse.ca/experience (click on any photo gallery)

**Comment by ronaldohoch** [comment]
Nice, thank you!
